bridge-adapters/kinetic-bridgehub-adapter-generic-rest/src/main/java/com/kineticdata/bridgehub/adapter/generic/rest/GenericRestAdapter.java

```diff
@@ -0,0 +1,416 @@
+package com.kineticdata.bridgehub.adapter.generic.rest;
+
+import com.jayway.jsonpath.DocumentContext;
+import com.jayway.jsonpath.JsonPath;
+import com.jayway.jsonpath.JsonPathException;
+import com.kineticdata.bridgehub.adapter.BridgeAdapter;
+import com.kineticdata.bridgehub.adapter.BridgeError;
+import com.kineticdata.bridgehub.adapter.BridgeRequest;
+import com.kineticdata.bridgehub.adapter.BridgeUtils;
+import com.kineticdata.bridgehub.adapter.Count;
+import com.kineticdata.bridgehub.adapter.Record;
+import com.kineticdata.bridgehub.adapter.RecordList;
+import com.kineticdata.commons.v1.config.ConfigurableProperty;
+import com.kineticdata.commons.v1.config.ConfigurablePropertyMap;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import net.minidev.json.JSONArray;
+import net.minidev.json.JSONObject;
+import org.apache.http.NameValuePair;
+import org.apache.http.client.utils.URLEncodedUtils;
+import org.apache.http.message.BasicNameValuePair;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class GenericRestAdapter implements BridgeAdapter {
+    /*----------------------------------------------------------------------------------------------
+     * CONSTRUCTOR
+     *--------------------------------------------------------------------------------------------*/
+    public GenericRestAdapter () {
+        // Parse the query and exchange out any parameters with their parameter 
+        // values. ie. change the query username=<%=parameter["Username"]%> to
+        // username=test.user where parameter["Username"]=test.user
+        this.parser = new GenericRestQualificationParser();
+    }
+
+    
+    /*----------------------------------------------------------------------------------------------
+     * PROPERTIES
+     *--------------------------------------------------------------------------------------------*/
+
+    /** Defines the adapter display name */
+    public static final String NAME = "Generic Rest Bridge";
```

Comment:
Is the convention we have been using for other adapters?

Reply:
Yes, all bridge adapters have the name of the adapter followed by Bridge.